src/state/CallViewModel.ts

```diff
@@ -668,8 +668,11 @@ export class CallViewModel extends ViewModel {
                             (
                               localUserMedia?.vm as LocalUserMediaViewModel
                             ).alwaysShow.pipe(
-                              map((alwaysShow) =>
-                                alwaysShow ? localUserMedia : undefined,
+                              map(
+                                (alwaysShow) =>
+                                  (alwaysShow
+                                    ? localUserMedia?.vm
+                                    : undefined) ?? null,
```

Comment:
Oh wow that one is new right?
Its the same issue as we ran into with the other case but I dont remember it from our test-fix-session.

Reply:
> Oh wow that one is new right? Its the same issue as we ran into with the other case but I dont remember it from our test-fix-session.

Yes. The compiler caught it, but our test cases did not.